packages/connect-query/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "./tsconfig.build.json",
   "compilerOptions": {
     "rootDir": ".",
+    "noEmit": true,
```

Comment:
Don't we want to emit here? 🤔

Reply:
Hmm, I think this might be an unnecessary change. Might have been switching contexts too much.

Reply:
Ah i see why i added this. It was a little weird, but there's an error in the this file (when looking at it in vscode): 

```
Cannot write file '/Users/paulsachs/code/connect-query-es/packages/connect-query/src/stable-hash.js' because it would overwrite input file.
```

Since this file is pretty much purely for editors to be able to default to something, I don't think adding `noEmit` harms anything but I agree, it is a little confusing.

Reply:
Oh, i guess tests also use this tsconfig.